feature/git/tests/test_df.py

```diff
@@ -4,7 +4,6 @@
 from datetime import datetime
 
 from dffml.df.types import Input, DataFlow
-from dffml.operation.output import GroupBy  # skipcq: PYL-W0611
```

Comment:
This import is used (although not obvious, via `sys.modules[__name__]`). It is needed, although, it might be helpful to include the `LGTM` ignore in the comment as well (the current comment is for deepsource.io to ignore)